### PR TITLE
(RE-4873) initial AIO packaging updates

### PR DESCRIPTION
**Pull request by heathseals**
This PR provides initial AIO/puppet-agent packaging updates to the CentOS, Ubuntu, and Debian PL packer boxes and bumps the version of the PE agent.  Where applicable, the ISO used during the provisioning process has been updated to use a full ISO instead of minimal ( to reduce network usage ).

### CODE REVIEW

File: manifests/modules/packer/manifests/vmtools.pp [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
   if ( $required_packages != undef ) {
     package { $required_packages:
-      ensure => installed,
-      before => File[ '/tmp/vmtools' ],
+      ensure     => installed,
+      before     => File[ '/tmp/vmtools' ],
```
Comment (ScottGarman) [inline manifests/modules/packer/manifests/vmtools.pp]: Why the change in  spacing here?

Comment (heathseals) [inline manifests/modules/packer/manifests/vmtools.pp]: they were aligned with another option that ended up not being required.  i'll update shortly.

File: templates/centos-6.6/x86_64.virtualbox.base.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
     {
       "template_name": "centos-6.6-x86_64",
       "template_os": "RedHat_64",
-
-      "iso_url": "http://mirrors.cat.pdx.edu/centos/6.6/isos/x86_64/CentOS-6.6-x86_64-netinstall.iso",
-      "iso_checksum": "2342bc5ea72fc028972aa0f21cee28b4",
+
```
Comment (ScottGarman) [inline templates/centos-6.6/x86_64.virtualbox.base.json]: Should this just be a newline? It looks to me like there are spaces added here.

File: templates/centos-6.6/x86_64.vmware.base.json [hunk 3]
```diff
@@ -4,9 +4,9 @@
     {
       "template_name": "centos-6.6-x86_64",
       "template_os": "rhel6-64",
-
-      "iso_url": "http://mirrors.cat.pdx.edu/centos/6.6/isos/x86_64/CentOS-6.6-x86_64-netinstall.iso",
-      "iso_checksum": "2342bc5ea72fc028972aa0f21cee28b4",
+
```
Comment (ScottGarman) [inline templates/centos-6.6/x86_64.vmware.base.json]: Same comment wrt spacing.

**Comment by ScottGarman** [comment]
I'm :+1:  other than the few really minor nit picks about whitespace.

File: templates/centos-6.6/x86_64.virtualbox.base.json (hunk 2 above)
Comment (heathseals) [inline templates/centos-6.6/x86_64.virtualbox.base.json]: not following.  can you elaborate?

Comment (ScottGarman) [inline templates/centos-6.6/x86_64.virtualbox.base.json]: Is line 7 just a newline, or are there spaces/tabs in it? If it's just a newline, then you can ignore my comment.

Comment (heathseals) [inline templates/centos-6.6/x86_64.virtualbox.base.json]: just a newline.

**Comment by stahnma** [comment]
I can't tell from the diff, but do we apt-get upgrade/yum upgrade after we install from the ISO? This would help for things like heartbleed and POODLEs and such.

**Comment by heathseals** [comment]
@stahnma not consistently across all the distributions.  that's something i noticed during this refresh and have a note to ticket and correct.

**Comment by stahnma** [comment]
Cool.
